addons/l10n_ie/data/tax_report-ie.xml

```diff
@@ -0,0 +1,117 @@
+                        <field name="label">balance</field>
+                        <field name="engine">aggregation</field>
+                        <field name="formula">l10n_ie_tr_T2.balance - l10n_ie_tr_T1.balance</field>
+                        <field name="subformula">if_above(EUR(0))</field>
+                    </record>
+                </field>
+            </record>
+            <record id="l10n_ie_tr_E1" model="account.report.line">
+                <field name="name">E1: intra-EU supplies of goods</field>
+                <field name="code">l10n_ie_tr_E1</field>
+                <field name="expression_ids">
+                    <record id="l10n_ie_tr_E1_tag" model="account.report.expression">
+                        <field name="label">balance</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">E1</field>
+                    </record>
+                </field>
+            </record>
+            <record id="l10n_ie_tr_E2" model="account.report.line">
+                <field name="name">E2: intra-EU acquisitions of goods</field>
+                <field name="code">l10n_ie_tr_E2</field>
+                <field name="expression_ids">
+                    <record id="l10n_ie_tr_E2_tag" model="account.report.expression">
+                        <field name="label">balance</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">E2</field>
+                    </record>
+                </field>
+            </record>
+            <record id="l10n_ie_tr_ES1" model="account.report.line">
+                <field name="name">ES1: intra-EU supply of service</field>
+                <field name="code">l10n_ie_tr_ES1</field>
+                <field name="expression_ids">
+                    <record id="l10n_ie_tr_ES1_tag" model="account.report.expression">
+                        <field name="label">balance</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">ES1</field>
+                    </record>
+                </field>
+            </record>
+            <record id="l10n_ie_tr_ES2" model="account.report.line">
+                <field name="name">ES2: intra-EU acquisition of services</field>
+                <field name="code">l10n_ie_tr_ES2</field>
+                <field name="expression_ids">
+                    <record id="l10n_ie_tr_ES2_tag" model="account.report.expression">
+                        <field name="label">balance</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">ES2</field>
+                    </record>
+                </field>
+            </record>
+            <record id="l10n_ie_tr_PA1" model="account.report.line">
+                <field name="name">PA1: postponed accounting</field>
```

Comment:
Any idea what this postponed accounting is?  Do we have a tax with that?

Reply:
It is related to Brexit basically. And yes we do have taxes for that https://github.com/odoo/odoo/pull/117312/files#diff-babf229f843259e2573a971112fa741af8d10249cd2e34839350845fdfbec869R192-R197